packages/astro/src/types/public/config.ts

```diff
@@ -95,6 +95,26 @@ export type ServerConfig = {
 	 * ```
 	 */
 	open?: string | boolean;
+
+	/**
+	 * @name server.allowedHosts
+	 * @type {string[] | true}
+	 * @default `false`
```

Comment:
Wouldn't the default be [an empty array](https://github.com/withastro/astro/blob/84d8fcf5df348bf988fdc6da09de141f88568534/packages/astro/src/core/config/schema.ts#L80) instead?
```suggestion
	 * @default `[]`
```